benchmarks/spack/csd3-cascadelake/compute-node/spack.yaml

```diff
@@ -95,20 +95,20 @@ spack:
         f77: /usr/local/software/master/gcc/9/bin/gfortran
         fc: /usr/local/software/master/gcc/9/bin/gfortran
       flags: {}
-      operating_system: scientific7
+      operating_system: centos7
       target: x86_64
       modules: []
       environment: {}
       extra_rpaths: []
   - compiler:
       spec: gcc@11.2.0
       paths:
-        cc: /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gcc
-        cxx: /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/g++
-        f77: /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran
-        fc: /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran
+        cc: /usr/local/software/spack/archive2/linux-centos7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gcc
+        cxx: /usr/local/software/spack/archive2/linux-centos7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/g++
+        f77: /usr/local/software/spack/archive2/linux-centos7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran
+        fc: /usr/local/software/spack/archive2/linux-centos7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran
```

Comment:
These paths don't exist.

Reply:
Well spotted! I've updated the paths

Seems like they've updated the OS from `scientific7` to `centos7` but they've moved their spack archive from `linux-centos7-skylake_avx512` to `linux-scientific7-skylake_avx512`? That makes no sense to me :confused:

Reply:
Sounds to me like they didn't move that directory at all :sweat_smile:

Is there any other GCC 11 available in the system not from a Spack installation?  But it shouldn't matter much, GCC can still generate code for the different ISAs.

Reply:
Ah, wait, did I automatically replace it in the pathname :facepalm:

Reply:
The `gcc/11` module points there, I don't know where else to look :shrug: @ma595?

```
[dc-kosk1@login-p-1 ~]$ module av gcc/11

---------------------------- /usr/local/software/modulefiles ----------------------------
gcc/11
[dc-kosk1@login-p-1 ~]$ module show gcc/11
-------------------------------------------------------------------
/usr/local/software/modulefiles/gcc/11:

prepend-path	 PATH /usr/local/software/master/gcc/11/bin 
prepend-path	 MANPATH /usr/local/software/master/gcc/11/share/man 
prepend-path	 LD_LIBRARY_PATH /usr/local/software/master/gcc/11/lib 
prepend-path	 LIBRARY_PATH /usr/local/software/master/gcc/11/lib 
prepend-path	 LD_LIBRARY_PATH /usr/local/software/master/gcc/11/lib64 
prepend-path	 LIBRARY_PATH /usr/local/software/master/gcc/11/lib64 
prepend-path	 C_INCLUDE_PATH /usr/local/software/master/gcc/11/include 
prepend-path	 CPLUS_INCLUDE_PATH /usr/local/software/master/gcc/11/include 
prepend-path	 INCLUDE /usr/local/software/master/gcc/11/include 
prepend-path	 PKG_CONFIG_PATH /usr/local/software/master/gcc/11/lib/pkgconfig 
prepend-path	 CMAKE_PREFIX_PATH /usr/local/software/master/gcc/11/ 
setenv		 CC /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gcc 
setenv		 CXX /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/g++ 
setenv		 FC /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran 
setenv		 F77 /usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gfortran 
-------------------------------------------------------------------
```

Reply:
Okay, there's links in `/usr/local/software/master/gcc/11/bin`, should we use those?

Reply:
As far as I can see those binaries point to the spack path
```console
$ realpath /usr/local/software/master/gcc/11/bin/gcc
/usr/local/software/spack/archive2/linux-scientific7-skylake_avx512/gcc-9.3.0/gcc-11.2.0-do3vmb6iub2hwfop75wwnhbsk3s2ucpi/bin/gcc
```
but probably it's nicer to use the path `/usr/local/software/master/gcc/11/bin`, which hopefully will be updated by sysadmins if necessary, instead of hardcoding the path with the spack spec slug.